kitsune/sumo/email_utils.py

```diff
@@ -31,6 +34,9 @@ def safe_translation(f):
     exception indicative of a bad localization of a string, try again in
     `settings.WIKI_DEFAULT_LANGUAGE`.
 
+    The translation system will be manipulated for each call to make normal
+    calls to `gettext` and friends use the appropriate language.
```

Comment:
Oops, I included too many words here, and it also confusing. How does this sound? "The translation system will be manipulated so that calls to `gettext` and friends will use the appropriate language."